lib/influxdb.coffee

```diff
@@ -1,78 +1,95 @@
-request = require('request')
-dgram = require('dgram')
+request = require "request"
+dgram = require "dgram"
 
 class Client
   constructor: (@config={}, @logger) ->
     do @init
 
   init: ->
-    useUDP = @config.get('influxdb.use_udp').get() ? false
+    useUDP = @config.get("influxdb.use_udp").get() ? false
 
     @send = if useUDP then @sendUDP() else @sendHTTP()
 
   write: (metrics) ->
-    @send @metricsJson metrics
+    @send @formatMetrics metrics
 
   sendHTTP: ->
-    version = @config.get('influxdb.version').get() ? '0.9'
-    host = @config.get('influxdb.host').get() ? 'localhost'
-    port = @config.get('influxdb.port').get() ? 8086
-    database = @config.get('influxdb.database').get() ? 'bucky'
-    username = @config.get('influxdb.username').get() ? 'root'
-    password = @config.get('influxdb.password').get() ? 'root'
+    version = @config.get("influxdb.version").get() ? "0.9"
```

Comment:
There was just a lot of mixing between single and double quotes, I wanted to make the quotes more uniform, I just picked one over the other.